### DISCUSSION TITLE
OneDrive存储在上传时仍走alist服务器的流量

### DISCUSSION THREAD

**Discussion by NikoCat233** [AlistGo/alist#7116] [Q&A] [answered]
Onedrive存储在下载时默认给的是onedrive的接口链接
但是上传时均先传到alist服务器，再由alist传到onedrive
请问Alist是没有直接调用onedrive上传接口的功能吗

**Comment by anwen-anyi** [comment, marked as answer]
所有的上传都是 `先传到搭建AList的设备\服务器，再由AList传到驱动终端`
